src/App.js

```diff
@@ -1,10 +1,19 @@
-import Navigation from './layout/navigation/Sidebar';
+import { Route, Routes } from 'react-router-dom';
+import { Provider } from 'react-redux';
 import 'bootstrap/dist/css/bootstrap.min.css';
+import Home from './components/Home/Home';
+import store from './redux/configureStore';
+import Navigation from './layout/navigation/Sidebar';
 
 function App() {
   return (
     <div className="App">
-      <Navigation />
+      <Provider store={store}>
```

Comment:
The `App` component is already wrapped inside Provider in `index.js`. I suggest you remove it from here.